src/analysis/normed_space/spectrum.lean

```diff
@@ -131,6 +140,137 @@ end
+  differentiable_on 𝕜 (λ z : 𝕜, ring.inverse (1 - z • a)) (metric.closed_ball 0 r) :=
+begin
+  intros z z_mem,
+  apply differentiable_at.differentiable_within_at,
+  have hu : is_unit (1 - z • a),
+  { refine is_unit_one_sub_smul_of_lt_inv_radius (lt_of_le_of_lt (coe_mono _) hr),
+    simpa only [norm_to_nnreal, real.to_nnreal_coe]
+      using real.to_nnreal_mono (mem_closed_ball_zero_iff.mp z_mem) },
+  have H₁ : differentiable 𝕜 (λ w : 𝕜, 1 - w • a) := (differentiable_id.smul_const a).const_sub 1,
+  exact differentiable_at.comp z (differentiable_at_inverse hu.unit) (H₁.differentiable_at),
+end
+
+end one_sub_smul
+
+section gelfand_formula
+
+open filter ennreal continuous_multilinear_map
+open_locale topological_space
+
+/- the assumption below that `A` be second countable is a technical limitation due to
+the current implementation of Bochner integrals in mathlib. Once this is changed, we
+will be able to remove that hypothesis. -/
+variables
+[normed_ring A] [normed_algebra ℂ A] [complete_space A]
+[measurable_space A] [borel_space A] [topological_space.second_countable_topology A]
+
+/-- The `limsup` relationship for the spectral radius used to prove `spectrum.gelfand_formula`. -/
+lemma limsup_pow_nnnorm_pow_one_div_le_spectral_radius (a : A) :
+  limsup at_top (λ n : ℕ, ↑∥a ^ n∥₊ ^ (1 / n : ℝ)) ≤ spectral_radius ℂ a :=
+begin
+  refine ennreal.inv_le_inv.mp (le_of_forall_pos_nnreal_lt (λ r r_pos r_lt, _)),
+  simp_rw [inv_limsup, ←one_div],
+  let p : formal_multilinear_series ℂ ℂ A :=
+    λ n, continuous_multilinear_map.mk_pi_field ℂ (fin n) (a ^ n),
+  suffices h : (r : ℝ≥0∞) ≤ p.radius,
+  { convert h,
+    simp only [p.radius_eq_liminf, ←norm_to_nnreal, norm_mk_pi_field],
+    refine congr_arg _ (funext (λ n, congr_arg _ _)),
+    rw [norm_to_nnreal, ennreal.coe_rpow_def (∥a ^ n∥₊) (1 / n : ℝ), if_neg],
+    exact λ ha, by linarith [ha.2, (one_div_nonneg.mpr n.cast_nonneg : 0 ≤ (1 / n : ℝ))], },
+  { have H₁ := (differentiable_on_inverse_one_sub_smul r_lt).has_fpower_series_on_ball r_pos,
+    exact ((has_fpower_series_on_ball_inverse_one_sub_smul ℂ a).exchange_radius H₁).r_le, }
+end
+
+/-- **Gelfand's formula**: Given an element `a : A` of a complex Banach algebra, the
+`spectral_radius` of `a` is the limit of the sequence `∥a ^ n∥₊ ^ (1 / n)` -/
+theorem pow_nnnorm_pow_one_div_tendsto_nhds_spectral_radius (a : A) :
+  tendsto (λ n : ℕ, ((∥a ^ n∥₊ ^ (1 / n : ℝ)) : ℝ≥0∞)) at_top (𝓝 (spectral_radius ℂ a)) :=
+begin
+  refine tendsto_of_le_liminf_of_limsup_le _ _ (by apply_auto_param) (by apply_auto_param),
+  { rw [←liminf_nat_add _ 1, liminf_eq_supr_infi_of_nat],
+    refine le_trans _ (le_supr _ 0),
+    exact le_binfi (λ i hi, spectral_radius_le_pow_nnnorm_pow_one_div ℂ a i) },
+  { exact limsup_pow_nnnorm_pow_one_div_le_spectral_radius a },
+end
+
+/-- **Gelfand's formula**: This is the same as
+`spectrum.pow_nnnorm_pow_one_div_tendsto_nhds_spectral_radius` except phrased in terms of `norm`
+instead of `nnnorm`. -/
```

Comment:
I think just copying the docstring from the other one would make sense, since that's what shows up when you hover your mouse over the theorem in VSCode and things like that. And then saying it's the same thing as the other one would just go in a regular comment (i.e. not a docstring).